.gitignore

```diff
@@ -29,4 +29,6 @@ local-chart/
 # word counts
 /counts
 
-
+# Hub Chart generated dependencies
+hub-chart/Chart.lock
```

Comment:
What is `hub-chart/Chart.lock` ?

Reply:
in the repo, hub-chart does not contain the dependency (kflex), which is pulled by go releaser before packaging the chart for the release. during that process 2 things are created:
1. Chart.lock
2. charts/....
During my local testing I obviously need the dependency... and I made the mistake before to push them to the git repo with commits... I added these lines to prevent that mistake.

Reply:
Thanks for the explanation.